pkg/drivers/kic/oci/cli_runner.go

```diff
@@ -63,8 +64,24 @@ func (rr RunResult) Output() string {
 	return sb.String()
 }
 
+// PrefixCmd adds any needed prefix (such as sudo) to the command
+func PrefixCmd(cmd *exec.Cmd) *exec.Cmd {
+	if cmd.Args[0] == Podman && runtime.GOOS == "linux" { // want sudo when not running podman-remote
```

Comment:
This will make a good insertion point for folks who want to add support for handling `rootless`. Thank you!